pkg/pdp/service/piece_upload.go

```diff
@@ -127,6 +127,21 @@ func (p *PDPService) UploadPiece(ctx context.Context, pieceUpload types.PieceUpl
 	}
 	log.Infow("computed piece commp", "request", pieceUpload, "commp", pieceCIDComputed.String())
 
+	// convert to a pieceCIDV2
+	pieceCIDV2, paddedSize, err := asPieceCIDv2(pieceCIDComputed.String(), uint64(upload.CheckSize))
```

Comment:
I would like to get to a point in the code where we do not have a distinction between v1 and v2 and we are just always using v2.